docs/python_support/reference.md

```diff
@@ -17,7 +17,7 @@
 The publicly available `openvino/model_server` image on Docker Hub supports Python, but does not come with external modules installed. If Python is all you need then you can use the public image without modification. Otherwise, you will need to extend the public image with additional layers that install any modules required for your Python code to run. For example, let's say your code requires numpy.
 In that case, your Dockerfile may look like this:
 
-```dockerfile
```

Comment:
Why it needs to be removed?

Reply:
Is it something that will be fixed in the future? It looks a bit better in markdown with that, so I'd prefer it to stay in the long term.